emote.py

```diff
@@ -14,11 +14,15 @@ def init(self):
 		if self.path is not None:
 			try:
 				if not os.path.isfile(self.path):
-					open(self.path, 'w+')
+					with open(self.path, 'w+') as f:
+						json.dump(self.emotes, f)
 
 				else:
-					self.emotes = json.loads(open(self.path, 'r').read())
+					with open(self.path, 'r') as f:
+						self.emotes = json.loads(f.read())
 
+			except (JSONDecodeError):
```

Comment:
This should be fine, I wasn't getting any errors for this. Just curious that in GitHub and in vim the exception isn't 'recognized' (?) like the IOError and OSError.